internal/services/contracts_events_consumer.go

```diff
@@ -123,6 +124,8 @@ func (c *ContractsEventsConsumer) Process(ctx context.Context, event *shared.Clo
 			return c.handleNewDcnNode(ctx, &data)
 		case NewExpiration:
 			return c.handleNewDCNExpiration(ctx, &data)
+		case NameChanged:
```

Comment:
Bigger problem: this event does not come from the DCN registry, it comes from the resolver. The registry does not know about names.

Reply:
So in the test, using the addresses from prod, the NameChanged event would come from [0x60627326F55054Ea448e0a7BC750785bD65EF757](https://polygonscan.com/address/0x60627326F55054Ea448e0a7BC750785bD65EF757#events)

https://polygonscan.com/address/0x60627326F55054Ea448e0a7BC750785bD65EF757#events

Reply:
I will try to get this running locally so you can see for yourself. No promises

Reply:
Registry events, for comparison

https://polygonscan.com/address/0xE9F4dfE02f895DC17E2e146e578873c9095bA293#events

Reply:
In general I think when writing these it’s good to use the “test data” that we have, namely the live deployments 

https://github.com/DIMO-Network/web3-deploys